### PR TITLE
Reminder functionality

### PR DESCRIPTION
**Pull request by Morvin-Ian**

### CODE REVIEW

File: app/urls_v2.py
Comment (WawinyEdwin) [inline app/urls_v2.py:1]: Why do we have change again?

File: app/helpers.py [hunk 1]
```diff
@@ -793,3 +794,49 @@ def check_last_finalizer(user, user_type, process)->bool:
...
+
+    
+def create_reminder(process, interval, members):
+    for member in members:
+        member = member.get("member")
+        if member:
+            ProcessReminder.objects.create(
+                process_id = process["_id"], 
+                step_finalizer = member,
+                email = "morvinian@gmail.com",
```
Comment (WawinyEdwin) [inline app/helpers.py:828]: Why are we sending this to your email?

File: app/constants.py [hunk 2]
```diff
@@ -270,21 +270,186 @@
...
+                            </p>
+                        </div>
 
+                        <div>
+                            <p>Best regards,</br>Workflow AI team</p>
+                        </div> 
+                </section>
+            </main>
+            <footer>
+                <a href="https://www.uxlivinglab.org/">Dowell Workflow AI</a>
```
Comment (WawinyEdwin) [inline app/constants.py:357]: This should be link to our workflowai client product,

File: app/constants.py [hunk 3]
```diff
@@ -270,21 +270,186 @@
...
+                <p>&copy; 2023 - All rights reserved.</p>
```
Comment (WawinyEdwin) [inline app/constants.py:358]: Add the Ux Living lab text here and give it the uxlivinglab.org link, For both templates

File: app/constants.py [hunk 4]
```diff
@@ -270,21 +270,186 @@
...
+        </style>
+    </head>
+    <body>
+        <div class="container">
+            <header>
+                <img src="https://dowellfileuploader.uxlivinglab.online/hr/logo-2-min-min.png" height="140px" width="140px" alt="DoWell Logo">
+            </header>
+            <main>
+                <section>
+                    <p>Dear User,</p>
```
Comment (WawinyEdwin) [inline app/constants.py:438]: Is it possible to make it more personalized, Like use a real username instead of a generic `User`

**Review by WawinyEdwin — CHANGES_REQUESTED** [review]
Please check on the given recommendations

File: app/constants.py (hunk 3 above)
Comment (Morvin-Ian) [reply app/constants.py:358]: Done

**Review by WawinyEdwin — CHANGES_REQUESTED** [review]
python runtime files .pyc are not supposed to be in this PR?? 
Is this a mistake

**Pull request closed without merging**